tests/Unit/API/REST/V3/Routes/Donors/GetDonorRouteTest.php

```diff
@@ -314,10 +314,9 @@ public function testGetDonorShouldRedactAnonymousDonor()
         $data = $response->get_data();
 
         $this->assertEquals(200, $status);
-        $this->assertEquals(0, $data['id']);
 
         $anonymousDataRedacted = [
-            //'id', // This property is Checked above...
```

Comment:
When the data is redacted, the ID should be zero, so we also need to rollback this logic.